app/views/hosts/_form.html.erb

```diff
@@ -22,6 +23,10 @@
       <% end %>
       <li><a href="#params" id='params-tab' data-url='<%= current_parameters_hosts_path %>' data-url2='<%= puppetclass_parameters_hosts_path %>' data-toggle="tab"><%= _('Parameters') %></a></li>
       <li><a href="#info" data-toggle="tab"><%= _('Additional Information') %></a></li>
+
+      <% @tabs.keys.each do |tab| %>
```

Comment:
then add "default aspects", this idea is very good IMHO, and now that it only adds clutter it misses the point. I might want to remove parameters altogether for hosts which are launched from a ready AMI (thus, they don't have a CM at all), I might decide to leave networking to the cloud platform, etc. this feature can allow a lot of flexibility, I don't see why we should limit it to new stuff.